libs/shared/src/lib/services/map/map-controls.service.ts

```diff
@@ -209,12 +212,91 @@ export class MapControlsService {
     if (this.fullscreenControl) {
       map.removeControl(this.fullscreenControl);
     }
-    this.fullscreenControl = new (L.Control as any).Fullscreen({
-      title: {
-        false: this.translate.instant('common.viewFullscreen'),
-        true: this.translate.instant('common.exitFullscreen'),
-      },
-    });
+    this.fullscreenControl = new L.Control({ position: 'topleft' });
+    const viewScreenText = this.translate.instant(
+      'components.map.controls.fullScreen.viewFullscreen'
+    );
+    const exitScreenText = this.translate.instant(
+      'components.map.controls.fullScreen.exitFullscreen'
+    );
+    this.fullscreenControl.onAdd = () => {
+      const mapContainer = map.getContainer();
```

Comment:
```suggestion
      const mapContainer = map.getContainer()?.parentElement;
```
@AntoineRelief 
This would fix the first part of this one: https://github.com/ReliefApplications/ems-frontend/pull/2422#issuecomment-1973281590 and fullscreen would still work fine